code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -242,8 +242,9 @@
 	return TRUE
 
 /obj/machinery/door/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+	var/isonshuttle = istype(get_area(src), /area/shuttle)
```

Comment:
dunno how's the rule on macros, but might want to make it a helper(for future uses and all of that)